test/e2e/suites/e2e/e2e_suite_test.go

```diff
@@ -45,6 +47,7 @@ func init() {
 func TestE2E(t *testing.T) {
 	RegisterFailHandler(Fail)
 	RunSpecs(t, "capo-e2e")
+	ctrl.SetLogger(klog.Background())
```

Comment:
Umm now you add the logger *after* running the specs :sweat_smile:

Reply:
whoops